t/file-filecopied2.neon

```diff
@@ -0,0 +1,15 @@
+IMPORT file
+
+TRY 
+    file.copyOverwriteIfExists("README.md", "tmp/license-copy.txt")
```

Comment:
Why does this test use `README.md`, while all the rest of the related tests use `LICENSE.txt`? Be consistent, it make more sense to use the same file for all of them.